include/reach/plugins/boost_progress_console_logger.h

```diff
@@ -18,8 +18,7 @@
 
 #include <reach/interfaces/logger.h>
 
-#include <boost/shared_ptr.hpp>
-#include <boost/progress.hpp>
+#include <boost/timer/progress_display.hpp>
```

Comment:
I think we could probably do something like this to continue supporting `focal` but fix the deprecation problem

```c++
#if __has_include(<boost/timer/progress_display.hpp>)
#include <boost/timer/progress_display.hpp>
#else
#include <boost/progress.hpp>
#endif
```

Reply:
The namespace also changed, so the fix is not as easy, unfortunately